### ISSUE TITLE
"Cannot open store for class "actorbas1", missing file?" Papyrus.log explanation

### ISSUE THREAD

**Issue by ThisIsPatrick** [ShikyoKira/Project-New-Reign---Nemesis-Main#373]
Having recently started using Nemesis over FNIS I've noticed this line began to show up in my papyrus logs.
My understanding of this message is that it's most often benign. The leftovers in a plugin/script that's looking for a script file that no longer exists (deprecated) or a soft depencency for a mod that the user doesn't have installed.

Having never seen a script with a naming convention as _actorbas1_, I decided to look into it deeper.

Doing a Notepad++ search of the entire Nemesis directory for _actorbas1_ returned nothing. Looking through Mod Organizer to see if any new scripts were created after running Nemesis and I found _FNIS.pex_.

Opening and searching _FNIS.pex_ for _actorbas1_ shows it can be found inside.
I decided to decompile it through Champollion. Sure enough, there was _actorbas1_ inside the source script as well.

**Under the function set_AACondition:**
`String report = "ERROR: Nemesis Alternate Animation 3D not loaded (Mod: " + mod + ", Target: " + (curAct.GetBaseObject() as actorbas1) as String + ")"`

**Attempting to recompile the script, of course, gives an error referring to _actorbas1_:**
`F:\ModOrganizer\SkyrimSE\overwrite\scripts\FNIS.psc(59,123): cannot convert to unknown type actorbas1
F:\ModOrganizer\SkyrimSE\overwrite\scripts\FNIS.psc(59,123): cannot cast a form to a actorbas1, types are incompatible
No output generated for FNIS.psc, compilation failed.`

After looking through the Nemesis_engine directory to find -where- the source for this script is I could only find a compiled version. The odd part being: the one I found is already compiled and it has no issues being decompiled then compiled.

**_Nemesis_Engine\scripts\launcher\end\XPMSE\script_**
`String report = "ERROR: Nemesis Alternate Animation 3D not loaded (Mod: " + mod + ", Target: " + (curAct.GetBaseObject() as actorbase) as String + ")"`

Comparing _Nemesis_Engine\scripts\launcher\end\XPMSE\script_ to the one Nemesis generates (_FNIS.pex_) shows them to be identical, besides the one difference of _actorbase_ becoming _actorbas1_

What I still don't understand is why this happens.
There is no reference to actorbas1 in the GitHub directory or the release download, leading me to believe this isn't intended.
My understanding of the script is that nothing problematic will happen: the script simply won't output a Debug Trace/Notification/MessageBox if the line of code is hit.

**TL;DR:**
At some point when running Nemesis Unlimited Behavior Engine.exe the generated _FNIS.pex_, based off _Nemesis_Engine\scripts\launcher\end\XPMSE\script_, becomes corrupted.
The problem is so minor that it might as well be ignored.
Replacing the generated _FNIX.pex_ with the one in _Nemesis_Engine\scripts\launcher\end\XPMSE\script_ fixes it.

I spent more time writing this than it warranted.
